torch/testing/_internal/common_device_type.py

```diff
@@ -930,37 +940,25 @@ def instantiate_device_type_tests(
                     )
                 else:
                     device_type_test_class.instantiate_test(name, copy.deepcopy(test))
-            else:  # Ports non-test member
-                assert (
```

Comment:
so we don't want this assert anymore? what's the implication of no longer having the assert?

Reply:
The new logic only brings over things from `generic_test_class` -> `device_type_test_class` that the latter doesn't already have.

In practice, `device_type_test_class` has these things starting out:
```
mappingproxy({'__module__': 'torch.testing._internal.common_device_type', '__doc__': None, '_classSetupFailed': False, '_class_cleanups': []})
```

These are all fine for the new device-specific class except arguably the `__module__`, and that it is set below (see line 952). So this just skips bringing these over from the generic test class.